src/web/server/app/utils/queries.ts

```diff
@@ -0,0 +1,17 @@
+// Query names take the following form: <ACTION>_<TABLE NAME>_<COLUMNS YOU WANT
```

Comment:
You should include some form of documentation that explains why you have two identical tables.

Reply:
I can but it's probably not helpful to put it in the `queries.ts` file, it mainly just comes down to the way Cassandra stores information.